packages/commonwealth/client/scripts/views/pages/AdminPanel/RPCEndpointTask.tsx

```diff
@@ -167,42 +143,15 @@ const RPCEndpointTask = () => {
   const update = async () => {
     if (!communityLookupData?.id) return;
     try {
-      let nodeId = null;
-      if (chainNodeNotCreated) {
-        // Create Chain Node if not yet created
-        const res = await createChainNode({
-          url: rpcEndpoint,
-          name: rpcName,
-          bech32,
-          balance_type: balanceType,
-          eth_chain_id: ethChainId,
-          cosmos_chain_id: cosmosChainId,
-        });
-        nodeId = res.data.result.node_id;
-      } else {
-        await updateChainNode({
-          id: communityChainNode.id,
-          url: rpcEndpoint,
-          name: rpcName,
-          bech32,
-          balance_type: balanceType,
-          eth_chain_id: ethChainId,
-          cosmos_chain_id: cosmosChainId,
-        });
-        // @ts-expect-error <StrictNullChecks/>
-        nodeId = communityChainNode.id;
-      }
-
-      if (
-        Object.keys(communityLookupData || {}).length > 0 &&
-        communityInfoValueValidated
-      ) {
-        await updateCommunity({
-          communityId: communityLookupData?.id,
-          chainNodeId: nodeId ?? communityChainNode?.id?.toString(),
-          type: ChainType.Chain,
-        });
-      }
+      await updateChainNode({
```

Comment:
The form is only for creating a chain node not updating. Updating has dangerous implications (removing a private Alchemy chain node).